supertokens_python/framework/fastapi/fastapi_request.py

```diff
@@ -60,7 +60,11 @@ def set_session_as_none(self):
         self.request.state.supertokens = None
 
     def get_path(self) -> str:
-        return self.request.url.path
+        root_path = self.request.scope.get("root_path")
+        url = self.request.url.path
+        # FastAPI seems buggy and it adds an extra root_path (if it matches):
+        # So we trim the extra root_path (from the left) from the url
```

Comment:
- "if it matches" -> matches what?
- what is the extra root path doesn't match? Then what will happen?